mlflow/models/evaluation/__init__.py

```diff
@@ -1,19 +1,17 @@
 from mlflow.models.evaluation.base import (
     ModelEvaluator,
-    EvaluationDataset,
     EvaluationResult,
-    EvaluationMetrics,
     EvaluationArtifact,
     evaluate,
     list_evaluators,
+    get_last_failed_evaluator,
 )
 
 __all__ = [
     "ModelEvaluator",
-    "EvaluationDataset",
     "EvaluationResult",
-    "EvaluationMetrics",
     "EvaluationArtifact",
     "evaluate",
     "list_evaluators",
+    "get_last_failed_evaluator",
```

Comment:
@dbczumar I suggest also export `get_last_failed_evaluator`, user can debug using it.

Reply:
I think we should wait to do this later, if necessary. Right now, the only evaluator that exists is the default evaluator. Ideally, error / warning messages should be enough to tell the user what went wrong. They shouldn't need to call an extra utility. Can we un-export this?